Library/Homebrew/tap.rb

```diff
@@ -221,10 +221,11 @@ def install(options = {})
 
     begin
       safe_system "git", *args
+      safe_system "brew", "readall", name
```

Comment:
I guess it doesn't hurt to check the aliases of the to-be-installed tap, too. Thus, I'd suggest:

```ruby
safe_system HOMEBREW_BREW_FILE, "readall", "--aliases", name
```